tests/utils.py

```diff
@@ -89,3 +97,16 @@ def is_logged_in(service, memo={}):
         proc = run(['login', '--check', url])
         memo[url] = proc.returncode == 0
     return memo[url]
+
+
+def get_handmade_sample_cases(self, *, html: str) -> List[TestCase]:
```

Comment:
Please move this function to `tests/service_yukicoder.py` (or, a new file like `tests/implementation_utils.py`) because `tests/utils.py` means utilities for tests, not tests for utilities (sorry for the confusing file name :bow:)